fineract-provider/build.gradle

```diff
@@ -289,7 +307,6 @@ distributions {
         distributionBaseName = "apache-fineract-binary"
         contents {
             from bootJar
-            from war
```

Comment:
Out of interest, does the distribution task actually work for you? I was yesterday working on another PR (automate updating of all the dependent licenses into the distribution) but could not get distTar etc to do anything.

Reply:
You got me ;-) ... that one would need rework too if we want to maintain the WAR files. I have no sense how often the WAR files are still used. I'd argue that it's outdated and that those who want it could easily build from distribution sources. But that's just my 2 cents. If a wider audience still needs this then I'll figure something out.

Reply:
Sorry - I didn't mean we should distribute the WAR file - I don't think that's necessary. But when I tried to use distTar I could not get it to do anything at all - not even build a tar file with the bootJar in it. Does it work for you?

Reply:
Ah, that... it should do something... good catch, I'll look at that.

Reply:
... actually it still does something (even creates the SHA512 file). The command is:

```
./gradlew -Pfineract.release clean build binaryDistTar
```

My plan was to replace that stitched together process with https://jreleaser.org/ ... less moving parts, proper Gradle plugin.

Reply:
Ok great - let me try that command, I was assuming the build would automatically be taken care of as a dependency. Will update the README if it works.